src/qml/Mysthea/Pages/Rulebook.qml

```diff
@@ -7,6 +7,9 @@ import Mysthea.Theme 1.0
 Page {
     id: root
     title: qsTr("RULEBOOK")
+
+    objectName: "rulebookPage"
```

Comment:
If you declare the various pages' `objectName` in a separate QML Singleton `PageName` might as well use that instead of writing each page name.